tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/NUnit/ICompositeWorkItem.cs

```diff
@@ -2,19 +2,19 @@
 // Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
+#nullable enable
```

Comment:
I assume `Children` can't be null?

Reply:
I believed that, but I saw an old NUnit version that in an edge case it could, so I changed to be nullable in [3d98a45](https://github.com/DataDog/dd-trace-dotnet/pull/6374/commits/3d98a45a5bebdc0c7de9990b062d3e8eb506713d)